portality/view/doajservices.py

```diff
@@ -138,3 +141,19 @@ def undismiss_autocheck(autocheck_set_id, autocheck_id):
         abort(404)
     return make_response(json.dumps({"status": "success"}))
 
+
+@blueprint.route('/export/article/<article_id>')
```

Comment:
Can I suggest that we make the URL with the `ris` format in it, like `/export/article/<article_id>/<format>`

And then request it as `/export/article/1234/ris`

That way we can extend this without changing the API shape later, and it makes the URL a bit more meaningful